skt/runner.py

```diff
@@ -98,6 +102,7 @@ def dumpjunitresults(self, jobid, junit):
             (stdout, stderr) = bkr.communicate()
 
     def getconsolelog(self, jobid=None):
+        """Retrieves console log from a running beaker test."""
```

Comment:
Could you please keep the descriptions in imperative mood? I.e. write "Retrieve", not "Retrieves". This way, I think, it's easier to reason about functions and quicker to grasp the essence of what they do.